### ISSUE TITLE
[qt5-base] build failure

### ISSUE THREAD

**Issue by Kerndog73** [microsoft/vcpkg#10801]
**Host Environment**
- OS: Windows 10
- Compiler: Microsoft (R) C/C++ Optimizing Compiler Version 19.25.28612 for x64

**To Reproduce**
`./vcpkg install qt5-base[latest]:x64-windows`

**Failure logs** 
```
Building package qt5-base[core,latest]:x64-windows...
-- Figuring out qt target mkspec. Target arch x64
-- Target mkspec set to: win32-msvc
-- Host mkspec not set. Qt's own buildsystem will try to figure out the host system
-- Downloading http://download.qt.io/official_releases/qt/5.13/5.13.1/submodules/qtbase-everywhere-src-5.13.1.tar.xz...
-- Extracting source C:/src/vcpkg/downloads/qtbase-everywhere-src-5.13.1.tar.xz
-- Applying patch patches/winmain_pro.patch
-- Applying patch patches/windows_prf.patch
-- Applying patch patches/qt_app.patch
-- Applying patch patches/gui_configure.patch
-- Applying patch patches/icu.patch
-- Applying patch patches/Qt5BasicConfig.patch
CMake Error at scripts/cmake/vcpkg_apply_patches.cmake:51 (message):
  Applying patch failed.  Checking patch
  mkspecs/features/data/cmake/Qt5BasicConfig.cmake.in...

  Hunk #1 succeeded at 124 (offset 71 lines).

  error: while searching for:

      )



  !!IF !isEmpty(CMAKE_WINDOWS_BUILD)

  !!IF isEmpty(CMAKE_LIB_DIR_IS_ABSOLUTE)

      set(imported_implib \"${_qt5$${CMAKE_MODULE_NAME}_install_prefix}/$${CMAKE_LIB_DIR}${IMPLIB_LOCATION}\")

  !!ELSE

      set(imported_implib \"IMPORTED_IMPLIB_${Configuration}\" \"$${CMAKE_LIB_DIR}${IMPLIB_LOCATION}\")

  !!ENDIF



  error: patch failed: mkspecs/features/data/cmake/Qt5BasicConfig.cmake.in:69

  error: mkspecs/features/data/cmake/Qt5BasicConfig.cmake.in: patch does not
  apply

Call Stack (most recent call first):
  scripts/cmake/vcpkg_extract_source_archive_ex.cmake:134 (vcpkg_apply_patches)
  ports/qt5-base/cmake/qt_download_submodule.cmake:32 (vcpkg_extract_source_archive_ex)
  ports/qt5-base/portfile.cmake:35 (qt_download_submodule)
  scripts/ports.cmake:90 (include)


Error: Building package qt5-base:x64-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: qt5-base:x64-windows
  Vcpkg version: 2020.02.04-nohash

Additionally, attach any relevant sections from the log files above.
```

**Additional context**
I know this isn't really "context" but I asked for the `[latest]` version and got 5.13.1. Did I do something wrong? I was expecting 5.14.2 because that's the latest version of Qt.

I noticed that a substantial change was recently made to the qt5-base port. Could that have introduced the problem I'm having?

**Comment by JackBoosY** [comment]
Confirmed, @Neumann-A could you please fix this issue?

Thanks.

**Comment by Neumann-A** [comment]
@JackBoosY: https://github.com/microsoft/vcpkg/pull/9705#issuecomment-607201046 -> TL;DR merge #10644

@Kerndog73: If you want Qt 5.14.2 go for #10644

**Comment by Neumann-A** [comment]
@JackBoosY: should be closed since #10644 is merged
